marconi-sidechain/test/Spec/Golden/Cli/marconi-sidechain___help.help

```diff
@@ -1,23 +1,23 @@
 marconi - a lightweight customizable solution for indexing and querying the
 Cardano blockchain
 
-Usage: marconi-sidechain [--version] --socket-path FILE-PATH --db-dir DIR 
-                         [--http-port HTTP-PORT] 
+Usage: marconi-sidechain [--version] (-s|--socket-path FILE-PATH)
```

Comment:
That's why I love golden tests. I can clearly see the changes that were made in the CLI interface.